Composer/packages/ui-plugins/virtual-assistant-creation/src/fileTypes.d.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Could this not be done a .ts file and have the compiler generate the d.ts?

Reply:
unused file, removing